Test-network.md

```diff
@@ -1,12 +1,11 @@
 # Storj Test Network
 
-The Storj test network (storj-sdk) enables you to run all the components of the Storj platform (Satellite, Uplink client, and storage nodes) and test them on your local machine.
+The `storj-sim` tool enables you to run all the components of the Storj network (Satellite, Storage Nodes, Console and Gateway) and test them on your local machine.
 
 ![network components](assets/components.png)
 
-In every day Storj usage, the Satellite, Storage node, and Uplink are run
-separately, but for the test network, all three components are set up and run
-in the same process.
+In every day Storj usage, the Satellite, Storage Node, and Uplink are run
+on separate servers and computers, but for the test network, all of the components are running locally.
```

Comment:
change to "but for the purposes of the test network, all of the three components are run locally."

Reply:
There are more components and will probably change in the future.